app.js

```diff
@@ -52,7 +51,11 @@ app.get('/home', function(req, res) {
 
 app.get('/feature-list', function(req, res) {
 	//for the time being (this should be changed to exist in the DB)
-    res.json(info.features)
+    db.any('SELECT DISTINCT feature FROM annotation').then(data => {
+        res.json(data.map(f => f.feature));
+    }).catch(err => {
+        res.json("[]");
```

Comment:
If this is an empty list, the user won't be notified that there are no features but then won't be able to annotate any batches. Should we send an error message instead so they at least know what's wrong?